### PR TITLE
1158 confirm window as more customizable

### LINKED ISSUE #1158: Confirm component usage changes
FE PR: https://github.com/HSLdevcom/jore-map-ui/pull/1171

Blocked by #1018 

* Fix to use confirm instead of alert: "Sinulla on tallentamattomia muutoksia
* Make confirm window component more customizable, change openConfirm() params given as an object

Current params:
```
    @action
    public openConfirm = (
        content: React.ReactNode,
        onConfirm: () => void,
        onCancel?: () => void
    ) => {
```
Fix params as object and to take custom text to onCancel and onConfirm buttons, something like:
```
    @action
    public openConfirm = ({
        content: React.ReactNode,
        onConfirm: () => void,
        onCancel?: () => void,
        confirmButtonText?: string,
        cancelButtonText?: string
    }) => {
```
Also maybe add some wrapper to the confirm content by default that puts some padding there.

Reasoning for this task: opening stop popup shows promp "Sinulla on tallentamattomia muutoksia. Haluatko varmasti vaihtaa solmua?" Should show options "Peruuta, Kyllä" instead of "Peruuta" "Hyväksy"

### PR DESCRIPTION
**Pull request by Divinare**
Closes #1158 

As stated in the card, fixes here are:
* Fix to use confirm instead of alert: "Sinulla on tallentamattomia muutoksia
* Make confirm window component more customizable, change openConfirm() params given as an object

One more confirm prompt related task will be done in: #1170

### CODE REVIEW

File: src/components/map/layers/NetworkLayers.tsx [hunk 1]
```diff
@@ -261,13 +261,13 @@ class NetworkLayers extends Component<INetworkLayersProps> {
...
-                    {properties.soltunnus}? Tallentamattomat muutokset kumotaan.
-                </div>,
-                triggerNetworkNodeClick
-            );
+            this.props.confirmStore!.openConfirm({
+                content: `Sinulla on tallentamattomia muutoksia. Haluatko varmasti avata solmun ${
+                    properties.soltunnus
+                }? Tallentamattomat muutokset kumotaan.`,
+                onConfirm: triggerNetworkNodeClick,
+                confirmButtonText: 'Kyllä'
```
Comment (ahjyrkia) [inline src/components/map/layers/NetworkLayers.tsx]: Maybe "Kumoa" instead of "Kyllä"?

File: src/components/overlays/confirm.scss [hunk 2]
```diff
@@ -9,6 +9,12 @@
         justify-content: center;
         flex-direction: column;
         align-items: center;
+        text-align: center;
```
Comment (ahjyrkia) [inline src/components/overlays/confirm.scss]: The confirm window is so wide. Maybe add max-width of 700px or something here?

File: src/components/map/layers/NetworkLayers.tsx (hunk 1 above)
Comment (Divinare) [reply src/components/map/layers/NetworkLayers.tsx]: Haluatko varmasti avata solmun 1130182? --> "Kyllä" is better

**Comment by Divinare** [comment]
Review fixes done.

**Comment by ahjyrkia** [comment]
changes ok

**Pull request merged**